charts/k8s-infra/templates/_config.tpl

```diff
@@ -62,7 +65,12 @@ Build config file for daemonset OpenTelemetry Collector: OtelAgent
 
 {{- define "opentelemetry-collector.loggingExporterConfig" -}}
 exporters:
-  logging: {}
+  logging:
+    {{- with .Values.presets.loggingExporter }}
+    verbosity: {{ .verbosity }}
+    sampling_initial: {{ .samplingInitial }}
+    sampling_thereafter: {{ .samplingThereafter }}
+    {{- end }}
```

Comment:
Did you test this? I don't think this works since they are not defined as `samplingInitial` and `samplingThereafter`.

Reply:
oh, nice catch. I tested with verbosity but not the other two.
anyways, fixing it.

Reply:
done